spec/source_finder/source_file_globber_spec.rb

```diff
@@ -113,11 +114,11 @@
   end
 
   SOURCE_FILE_GLOB =
-    '{Rakefile,Dockerfile,{*,.*}.' \
-    '{rb,rake,gemspec,swift,cpp,c,java,py,clj,cljs,scala,js,' \
-    'yml,sh,json},{src,app,config,db,lib,test,spec,feature}/' \
-    '**/{*,.*}.{rb,rake,gemspec,swift,cpp,c,java,py,clj,cljs,scala,' \
-    'js,yml,sh,json}}'
+    '{Dockerfile,Rakefile,{*,.*}.' \
+    '{c,clj,cljs,cpp,gemspec,java,js,json,py,rake,rb,scala,sh,swift,' \
+    'yml},{app,config,db,feature,lib,spec,src,test,www}/' \
+    '**/{*,.*}.{c,clj,cljs,cpp,gemspec,java,js,json,py,rake,rb,scala,sh,' \
+    'swift,yml}}'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.